install/install.py

```diff
@@ -13,9 +13,7 @@
 
```

Comment:
The original blank lines are okay. No need to remove them. Only one comment below.